plugins/main/public/components/common/search-bar/search-bar-service.ts

```diff
@@ -111,7 +113,12 @@ export const search = async (
     .setField('filter', filters)
     .setField('query', query)
     .setField('sort', sortOrder)
-    .setField('size', pagination?.pageSize)
+    .setField(
```

Comment:
I think you should write the conditional before assigning its value to the size property, so it's easier to read. A comment explaining why can also be useful

Reply:
Applied in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6917/commits/704b342dbb5a48c7d037235c2c2eda2f84653980